contrib/drivers/drivers.go

```diff
@@ -136,7 +146,7 @@ func Clean(d VolumeDriver) VolumeDriver {
 	case *spectrumscale.Driver:
 		break
 	case *oceanstor.Driver:
-		break
+		return // No need to clean anything for oceanstor
```

Comment:
What happen, if i really mean to unset/clean the driver??

Reply:
oceanstor driver's unset operation just close the connection to back storage, cause we use the same global oceanstor driver for each call now, so there's no need to unset anymore. Instead, we want to keep the connection session active consistently.

Reply:
ok